Dockerfile.mysql

```diff
@@ -0,0 +1,13 @@
+FROM mysql:latest
+
+ENV MYSQL_ROOT_PASSWORD=1234
+ENV MYSQL_DATABASE=app_db
+ENV MYSQL_USER=app_user
+ENV MYSQL_PASSWORD=1234
```

Comment:
Using a simple password like '1234' for the database user is a significant security risk. Please update this to a strong, unique password and consider managing it through environment variables.